pkgs/top-level/release-lib.nix

```diff
@@ -34,7 +34,20 @@ let
     systems
     ;
 
-  pkgs = packageSet (recursiveUpdate { inherit system; config.allowUnsupportedSystem = true; } nixpkgsArgs);
+  allNixpkgsArgs = lib.recursiveUpdate
```

Comment:
This should cover all the release-*.nix files providing their own `nixpkgsArgs`.